app/build.gradle

```diff
@@ -58,12 +58,17 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    compileOptions {
+        targetCompatibility 1.8
+        sourceCompatibility 1.8
```

Comment:
does this still work with old versions of android, or does language version not matter?

Reply:
Yep, still works. Or at least works on the `4.4.something` Motorolla phones we have kicking around.

Reply:
We have a minimum supported SDK thing set in our library and it doesn't complain about it.